data/json/items/containers.json

```diff
@@ -272,6 +272,21 @@
     ],
     "qualities": [ [ "BOIL", 1 ] ]
   },
+  {
+    "id": "bottle_plastic_sealed",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "sealed plastic bottle" },
+    "copy-from": "bottle_plastic",
+    "description": "A watertight plastic bottle, holds 500 ml of liquid. This one has a tamper-evident cap and foil seal, which is still intact.",
```

Comment:
This description won't change, so it shouldn't mention that it's still sealed.

Reply:
Done - changed to "preserves content from rot until opened" from condiment bottle description.